modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/HiveCreateSTARJobs.pm

```diff
@@ -1,68 +1,69 @@
 =head1 LICENSE
  
- Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
```

Comment:
Could you briefly express the changes here? I am confused about how they are related to software paths changes